server/services/index.ts

```diff
@@ -5,9 +5,11 @@ import CommunityService from './communityService'
 import HmppsAuthClient from '../data/hmppsAuthClient'
 import TokenStore from '../data/tokenStore'
 import CaseloadService from './caseloadService'
+import PrisonApiClient from '../data/prisonApiClient'
 
 const hmppsAuthClient = new HmppsAuthClient(new TokenStore())
-const userService = new UserService(hmppsAuthClient)
+const prisonApiClient = new PrisonApiClient('no admin token required for user service')
```

Comment:
Putting this string as a parameter will actually result in the requests to prison API having this header?

`Authorization: Bearer no admin token required for user service`